.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -40,15 +40,15 @@ jobs:
       - name: "flake8 Style Checking"
         # language=bash
         run: |
-          pip install colorama flake8
+          pip install colorama flake8 flake8-bugbear
           python -m flake8 aiosmtpd setup.py housekeep.py release.py
       - name: "Docs Checking"
         # language=bash
         run: |
           pip install colorama pytest pytest-mock sphinx sphinx-autofixture sphinx_rtd_theme
-          pytest -v aiosmtpd/docs
-          sphinx-build --color -b html -d build/.doctree aiosmtpd/docs build/html
-          sphinx-build --color -b man  -d build/.doctree aiosmtpd/docs build/man
+          sphinx-build --color -b doctest -d build/.doctree aiosmtpd/docs build/doctest
```

Comment:
We need selective doctest (UnixSocketController doesn't work on Windows). pytest can't do that, but sphinx-build can.

A side benefit: No longer receive the DeprecationWarning warning :-)